repositories.gradle

```diff
@@ -8,7 +8,7 @@ repositories {
     }
     maven {
         name 'ic2'
-        url 'https://maven.ic2.player.to/'
+        url 'https://maven2.ic2.player.to/'
```

Comment:
You shouldn't replace the maven url, add it as a second line to be a mirror

Reply:
I was given an example that added a second url, and I still replaced instead of adding...

Fixed now.